app/controllers/searches_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SearchesController < ApplicationController
+  def index
```

Comment:
This should be on the items controller. No need to make a new controller or model for search.